interpretation/ww_head.py

```diff
@@ -166,14 +167,31 @@ def process(message, isGameMaster = False):
         # Says the user must be in a cc if they're not.
         if is_command(message,['info']):
             guild = message.channel.guild
-            owner_id = channel_get(message.channel.id,'owner')
-            owner_object = guild.get_member(int(owner_id))
+            try:
+                owner_id = channel_get(message.channel.id,'owner')
+            except:
+                return[Mailbox().respond('Sorry, but it doesn\'t look like you\'re in a CC! If you are, please alert a Game Master as soon as possible.')]
+            if owner_id != None:
+                owner_object = guild.get_member(int(owner_id))
             embed = Embed(color=0x00cdcd, title='Conspiracy Channel Info')
-            embed.add_field(name='Channel Owner', value='<@' + owner_id + '>')
+            if owner_object != None and owner_id != None:
+                embed.add_field(name='Channel Owner', value='<@' + owner_id + '>')
+                embed.set_thumbnail(url=owner_object.avatar_url)
+            elif owner_id == None:
+                return [Mailbox().respond('Sorry, but it doesn\'t look like you\'re in a CC! If you are, please alert a Game Master as soon as possible.')]
```

Comment:
Why are you creating an embed, adding unnecessary if-statements and configuring things if you could just put this instead of the try-except statement, to catch out anything that goes wrong?